spec/factories/case_information.rb

```diff
@@ -30,4 +30,8 @@
 
     crn { Faker::Alphanumeric.alpha(number: 10) }
   end
+
+  trait :parole_review_date do
+    parole_review_date { Date.new(2019, 01, 3).to_s }
+  end
```

Comment:
I was going to say that this probably doesn't need to be a trait, and could be just a regular field/attribute on the factory instead.

But seeing how the other fields have been moved into traits over in the `:sentence_details` factory, I'm sure there's a reason for it!